intro-HPC/ch_account.tex

```diff
@@ -345,7 +345,7 @@ \subsection{Welcome e-mail}
 Within one day, you should receive a Welcome e-mail with your VSC account details.
 
 \begin{flattext}
-Dear Subject Name Here,
+Dear (Subject Name Here),
```

Comment:
I'm not seeing the change. Did you alter and forget to push or did my remark not make sense?